tests/jobs/test_scheduler_job.py

```diff
@@ -3902,13 +3902,16 @@ def test_process_dags_queries_count(
 
             failures = []  # Collects assertion errors and report all of them at the end.
             message = "Expected {expected_count} query, but got {current_count} located at:"
-            for expected_query_count in expected_query_counts:
-                with create_session() as session:
-                    try:
-                        with assert_queries_count(expected_query_count, message):
-                            self.scheduler_job._do_scheduling(session)
-                    except AssertionError as e:
-                        failures.append(str(e))
+            # in Debian Bullseye for MSSQL one of the tests gets into additional loop
+            # More often, and we should increase the interval of fetching to avoid that
+            with conf_vars({('core', 'min_serialized_dag_fetch_interval'): '30'}):
```

Comment:
On L3889 We set the value to 100 (seconds), so this setting here actually _decreases_ the timeout, so this comment doesn't quite hold up.

Reply:
Right. That is strange. I think this might be a temporary effect. I will take a close look at the #21378 when we actually swtich.

Reply:
Yeah. Seem's that removing that change did not change anything in #21378 :)